### PR TITLE
Jamakase/account settings page

### LINKED ISSUE #4486: Airbyte Cloud: Settings page
## Tell us about the problem you're trying to solve
* Settings are going to be a little different for Airbyte Cloud than it is for the OSS version. Things like inviting users need to be included.

## Describe the solution you’d like
* [Wireframes](https://www.figma.com/file/etaOkOsnXMoKwbfhCD71kw/Web-app?node-id=26178%3A2589). Note that these aren't full mock ups yet. They give you an idea of what will be there, but styling isn't ready yet.

### LINKED ISSUE #5220: use the full width of the screen when displaying logs in the UI
## Tell us about the problem you're trying to solve
The window for reading logs is too small: 
Screen Shot 2021-08-04 at 11 02 17 PM

## Describe the solution you’d like
1. Allow the log window to take up 90 or 95% of the window size
Screen Shot 2021-08-04 at 11 01 46 PM
2. Disable the need for horizontal scrolling within the wide window 

A solution could potentially be to create a button which opens a full-screen modal with the logs

cc @jrhizor

### LINKED ISSUE #5322: Use Airbyte Styling on password reset page
## Tell us about the problem you're trying to solve
image (14)
When a user clicks the password reset link that we send in an email they end up on this page that is styled using firebase styling instead of Airbyte styling. We'd like it to look cohesive with the rest of the styles

### LINKED ISSUE #5446: Document how to use StoryBook for connector configurations
## Tell us about the problem you're trying to solve
@Jamakase added a cool tool called StoryBook to the FE code. It enables us to run just an individual react component. This is very helpful in the case where for a connector, we want to see what its configuration will look like in the UI. We can get a super fast iteration cycle by just loading it in story book without having to rebuild the world.

## Describe the solution you’d like
The ask for this issue is for @Jamakase to document how this can be used by other developers on the team.

@sherifnada - FYI. You'll think this is cool

### LINKED ISSUE #5447: Document (and maybe improve) how we inject "runtime" environment variables in the webapp
## Tell us about the problem you're trying to solve
* We have a way of injecting runtime environment variables into the webapp using nginx. It is a little hard to do though. We want to _at least_ document it to make it easy. This is really important because right now, we are setting env variables at build time. This is bad because it is easy to forget to set them properly at build time (as opposed to having the infrastructure inject the properly at runtime). Also it requires rebuilding the webapp 4 times to do a release (OSS, dev, staging, prod).
* Potentially we want to update how we do this injection to be easier. @Jamakase suggested injecting a json object via docker compose into nginx, instead of doing one env variable at a time. If he can do that cleanly via docker compose that would be great. Some open question of how to make that look good in docker compose.

## Question for the future
* We will want to host the webapp in a static bucket / CDN soon. That means we will _not_ have an nginx server sitting in front of it. How will we inject the env variables in that scenario. @davinchia @Jamakase wdyt? We can close this issue once we solve the bullet points above but I want to start this conversation now since it may be relevant to how we think about the implementation.

### PR DESCRIPTION
**Pull request by jamakase**
## What
Finalize accounts settings page. ( Closes airbytehq/airbyte#4486   )

Add instruction how to use storybook ( Closes https://github.com/airbytehq/airbyte/issues/5446 )
Document how to set env variables ( https://github.com/airbytehq/airbyte/issues/5447 )
Closes airbytehq/airbyte#5220 (full width log display)

### CODE REVIEW

**Review by jrhizor — COMMENTED** [review]
1. #4486
Looks good on the OSS version.

2. #5446
I followed the instructions here and they worked for me 👍 

3. #5447
Left some comments on this.

4. #5220
All good.

Something broke on the sources/destinations page. At my laptop display size (which is pretty common), the source/destination connector pages are wider than the screen:
Screen Shot 2021-08-26 at 12 03 50 AM

**Comment by cgardens** [comment]
Just tried out the storybook feature for looking at how a connector configuration would render. 😍 😍 😍 😍  so good.

File: airbyte-webapp/docs/HowTo-EnvVariables.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+### Static env variables
+
+The environment variables are embedded during the build time. Since our app is based on Create React App that produces a
+static HTML/CSS/JS bundle, it can’t possibly read them at runtime.
+
+Static env variables name should always start with `REACT_APP_`
+
+### Dynamic env variables
+
+Dynamic env variables in our cases are injected into app by nginx
```
[the remark this answers is not in the document]
Comment (Jamakase) [reply airbyte-webapp/docs/HowTo-EnvVariables.md:17]: Do you mean that we need to add comment into docker-compose and kube files or add few lines here saying that they are set in docker-compose and kube?

[the remark this answers is not in the document]
Comment (jrhizor) [reply airbyte-webapp/docs/HowTo-EnvVariables.md:17]: The latter.

**Review by jrhizor — APPROVED** [review]

**Pull request merged**